pkg/compute/deploy.go

```diff
@@ -291,6 +293,18 @@ func (c *DeployCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		return err
 	}
 
+	if c.Comment.WasSet {
+		_, err = c.Globals.Client.UpdateVersion(&fastly.UpdateVersionInput{
+			ServiceID:      serviceID,
+			ServiceVersion: version.Number,
+			Comment:        &c.Comment.Value,
+		})
+
+		if err != nil {
+			text.Error(out, "failed to set comment for version %d: %e", version.Number, err)
```

Comment:
If the use case for a comment is to annotate a release of a service/package (as per https://github.com/fastly/cli/issues/328) then I wonder if we should set the expectation to be that a failure to update the service with the given comment to result in the deploy flow to be stopped (i.e. don't then go ahead and activate the service).

/cc @acme what are your thoughts on this?

Reply:
I thought about this as well and ultimately decided that setting a comment wasn't critical, but that really depends on what makes most sense. It's no issue to change this to fail the flow instead of just logging the error.

Reply:
Thanks for your contribution! I feel that if a comment has been passed as an option, then it is a critical part of the new service version, much like a version control commit message.

Reply:
Thanks @acme 

@bbutkovic if you wouldn't mind making those changes please, and then I'll re-review. Many thanks!

Reply:
Makes sense, thank you for the input! I'll make the changes soon.